Hardcore.lua

```diff
@@ -2259,6 +2274,68 @@ function Hardcore:PrintBubbleHearthInfractions()
 	end
 end
 
+function Hardcore:GenerateVerificationStatusStrings()
+	local statusString = ""
+	local numDeaths = #Hardcore_Character.deaths
+	local perc = string.format( "tracked_time=%.1f%%", Hardcore_Character.tracked_played_percentage )
+	local numGaps = #Hardcore_Character.played_time_gap_warnings
+	local numTrades = #Hardcore_Character.trade_partners
+	local numBubs= #Hardcore_Character.bubble_hearth_incidents
+	local verdict = ""
+	local COLOR_WHITE = "|c00ffffff"
+	local reds = {}
+	local yellows = {}
+	local greens = {}
+	
+	-- Determine the end verdict
+	if numTrades <= 0 and numDeaths <= 0 and numBubs <= 0 and numGaps <= 0 and Hardcore_Character.tracked_played_percentage >= 95 then
```

Comment:
For another day, but it would be nice to have meta data for saving and these validity checks